src/components/views/settings/tabs/user/HelpUserSettingsTab.js

```diff
@@ -225,6 +225,15 @@ export default class HelpUserSettingsTab extends React.Component {
                                 {_t("Clear cache and reload")}
                             </AccessibleButton>
                         </div>
+                        {
+                            _t( "To submit a security-related bug report, please read the Matrix.org " +
+                                "<a>Security Disclosure Policy</a>.", {},
+                                {
+                                    'a': (sub) =>
+                                        <a href="https://matrix.org/security-disclosure-policy/"
+                                        rel='noreferrer noopener' target='_blank'>{sub}</a>,
```

Comment:
can this use double quotes like the line above it for consistency and my sanity :D

Reply:
yes